packages/forms/README.md

```diff
@@ -0,0 +1,32 @@
+# Forms
```

Comment:
Some of this information seems redundant with what we render in Storybook. I'd recommend we task @lsamimi with writing text input documentation